tutorials/Window.stopNavigation.md

```diff
@@ -2,9 +2,22 @@ Stops any current navigation the window is performing.
 
 # Example
 ```js
+<<<<<<< HEAD
```

Comment:
looks like merge artifacts, should I close this PR as well?

Reply:
NO its good pushed up the clean version